composer.json

```
@@ -20,10 +20,10 @@
   "require-dev": {
     "roave/security-advisories": "dev-latest",
     "phpunit/phpunit": "^9.6.7",
-    "vimeo/psalm": "^5.26",
+    "vimeo/psalm": "6.0",
```

Thanks for the upgrade, but the version constraint is too restrictive. Consider using a more flexible constraint like `^6.13`, which is [the latest version available](https://packagist.org/packages/vimeo/psalm#6.13.1), and still allows for the use of higher minor versions.

```suggestion
    "vimeo/psalm": "^6.13",
```